include/aspect/material_model/interface.h

```diff
@@ -274,6 +274,18 @@ namespace aspect
       std::vector<double> viscosities;
 
       /**
+       * Constitutive tensors at the given positions. This variable
+       * can be used to implement exotic rheologies such as anisotropic
+       * viscosity.
+       *
+       * @note The strain rate term in equation (1) of the manual will be
+       * multiplied by this tensor *and* the viscosity scalar (\\eta). If no
+       * value is assigned here the identity tensor will be used in its place.
+       * This leaves the isotropic constitutive law unchanged by default.
+       */
+      std::vector<SymmetricTensor<4,dim> > constitutive_tensors;
```

Comment:
I find the term `constitutive_tensors` too vague. How about `stress_strain_directors` to match the terminology you use in the manual? You could also reference the manual here.